code/iaas/allocator/pom.xml

```diff
@@ -28,5 +28,10 @@
             <artifactId>cattle-framework-jooq</artifactId>
             <version>${project.version}</version>
         </dependency>
+        <dependency>
+            <groupId>io.cattle</groupId>
```

Comment:
Son, can you double check that there are no circle references in mvn project? As service-discovery-service project has cattle-iaas-allocator and cattle-iaas-service-discovery-api dependencies (code/iaas/service-discovery/server/pom.xml)